src/SimpleAzure.Storage.HybridQueue/LoggingExtensions.cs

```diff
@@ -2,23 +2,9 @@
 
 namespace WorldDomination.SimpleAzure.Storage.HybridQueues;
 
-public static class LoggerExtensions
+internal static class LoggerExtensions
 {
-    public static IDisposable? BeginCustomScope<T>(this ILogger<T> logger, params (string Name, object? Value)[] scopeItems)

```

Comment:
Interesting .. I thought codecov said _the other logging function_ under this, was never called.

Can you explain what you've done here? i mean, i can see you've deleted this section .. but why this one when i thought this one should stay and the bottom one (ILogger .. not ILogger<T>) should go?

maybe i'm calling the wrong `BeginCustomScope` method from my other code?

Reply:
This method was identical to the other method. Since ILogger<T> implements ILogger, anything matching this method would also match the  one below. This could be useful if this method had done anything special with the ILogger<T>, or with the T, but it didn't.

Codecov would have said the other method was redundant, because everything matched this one, but it's better to target the lowest level type you need IMO

Reply:
Approved.